wheel_odometry:
  robot_lenght: 1.64
  robot_width: 1.06
  wheel_diameter: 0.48
  max_angle_left: 35
  max_angle_right: 30
  steer_bar_zero: 0
  